cidc_api/services/ingestion.py

```diff
@@ -301,6 +309,9 @@ def upload_assay():
         "url_mapping": url_mapping,
         "gcs_bucket": GOOGLE_UPLOAD_BUCKET,
     }
+    if schema_hint in prism.ASSAYS_WITH_EXTRA_METADATA:
```

Comment:
shouldn't this just rely on `files_with_extra_md` list being not empty? then we won't need this extra prism dependency (tighter coupling).